project/src/main/ui/chat/chat-history.gd

```diff
@@ -11,19 +11,30 @@ const CHAT_AGE_NEVER := 99999999
 ## Tracks which conversations the player has had with each creature. The value is a per-creature index which starts
 ## from 0 and increments with each conversation with that creature.
 ##
-## key: chat key
-## value: ordering of when the chat happened (smaller is older)
+## key: (String) chat key
+## value: (int) ordering of when the chat happened (smaller is older)
 var chat_history: Dictionary
 
-## Stores flags which can be stored/received from chatscript. Flags represent chat choices the player made, such as
-## the answer to a yes or no question. They are persisted alongside the player's chat history.
-var _flags: Dictionary
+## Phrases which can be stored/retrieved from chatscript. Phrases represent chat choices the player made, such as the
+## answer to a short answer question. They are persisted alongside the player's chat history.
+##
+## key: (String) phrase name
+## value: (String) phrase value
+var phrases: Dictionary
+
+## Flags which can be stored/retreived from chatscript. Flags represent chat choices the player made, such as the
+## answer to a yes or no question. They are persisted alongside the player's chat history.
+##
+## key: (String) flag name
+## value: (String) flag value
+var flags: Dictionary
```

Comment:
Move this up to preserve alphabetical order. (I think you reordered these when some of the fields were private.)

Reply:
Moved.